### PR TITLE
Feat choose payment modal 

### PR DESCRIPTION
**Pull request by shirmagen**

### CODE REVIEW

File: examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx [hunk 1]
```diff
@@ -56,44 +60,68 @@ const PaymentMethodPopup = ({
...
+          display: 'flex',
+          flexDirection: 'row',
+        }}
+        >
+          <Title style={{ right: 0 }}>{i18n.t('popups.choosePaymentMethod.title')}</Title>
+          <Button
+            style={{ height: 0, width: 0 }}
+            onPress={() => {
+              onCancel();
+              rideFlow
```
Comment (oriefrati) [inline examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx]: remove all styles, use styled component

File: examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx [hunk 2]
```diff
@@ -56,44 +60,68 @@ const PaymentMethodPopup = ({
...
+
+      <SummaryContainer style={{ maxHeight: 450 }}>
+        <View style={{
+          display: 'flex',
+          flexDirection: 'row',
+        }}
+        >
+          <Title style={{ right: 0 }}>{i18n.t('popups.choosePaymentMethod.title')}</Title>
+          <Button
+            style={{ height: 0, width: 0 }}
```
Comment (oriefrati) [inline examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx]: why 0,0?

File: examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx [hunk 3]
```diff
@@ -56,44 +60,68 @@ const PaymentMethodPopup = ({
...
+      isVisible={isVisible}
+    >
+
+
+      <SummaryContainer style={{ maxHeight: 450 }}>
+        <View style={{
+          display: 'flex',
+          flexDirection: 'row',
+        }}
+        >
```
Comment (oriefrati) [inline examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx]: styled component

File: examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx [hunk 4]
```diff
@@ -56,44 +60,68 @@ const PaymentMethodPopup = ({
...
+          <Button
+            style={{ height: 0, width: 0 }}
+            onPress={() => {
+              onCancel();
+              rideFlow
+                ? navigation.navigate(MAIN_ROUTES.HOME)
+                : navigation.navigate(MAIN_ROUTES.PAYMENT);
+            }}
+          >
+            <SvgIcon Svg={closeIcon} style={{ margin: 0, height: 2, width: 2 }} />
```
Comment (oriefrati) [inline examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx]: styled component

File: examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx [hunk 5]
```diff
@@ -56,44 +60,68 @@ const PaymentMethodPopup = ({
...
+
+          </Button>
+        </View>
+        <ScrollView style={{ maxHeight: 400, marginBottom: 50 }}>
```
Comment (oriefrati) [inline examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx]: styled component

File: examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx [hunk 6]
```diff
@@ -56,44 +60,68 @@ const PaymentMethodPopup = ({
...
+            </View>
+          </Container>
+        </ScrollView>
+        <Footer>
+          <FlexCont style={{ justifyContent: 'center' }}>
+            <RoundedButton
+              style={{ margin: 10 }}
+              type="confirm"
+              onPress={() => onSave()}
+            >
```
Comment (oriefrati) [inline examples/client/Locomotion/src/popups/ChoosePaymentMethod/index.tsx]: put inside container and add margin

File: examples/client/Locomotion/src/pages/Payments/credit-cards.tsx [hunk 7]
```diff
@@ -45,28 +45,31 @@ export default ({
...
-              <Section
-                title={i18n.t('payments.otherMethodsTitle')}
-                showChangeButton={false}
-                onPress={(paymentMethod: any) => navigate(MAIN_ROUTES.CARD_DETAILS,
-                  { paymentMethod })}
-                paymentMethods={usePayments.paymentMethods
-                  .filter(({ id }) => id !== defaultMethod.id)}
-              />
-            ) : undefined}
+            {usePayments.paymentMethods.length > 1
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/Payments/credit-cards.tsx:61]: const { paymentMethods } = usePayments

**Review by oriefrati — APPROVED** [review]

**Pull request merged**